docs/examples/widgets/radio_button.py

```diff
@@ -0,0 +1,26 @@
+from textual.app import App, ComposeResult
+from textual.widgets import RadioButton, RadioSet
+
+
+class RadioChoicesApp(App[None]):
+    CSS_PATH = "radio_button.css"
+
+    def compose(self) -> ComposeResult:
+        yield RadioSet(
```

Comment:
You can use the new context manager syntax now if you like, since its in the docs.

Reply:
happy-dance.gif